cohttp-lwt-unix/src/server.mli

```diff
@@ -34,5 +34,6 @@ val create :
   ?backlog:int ->
   ?stop:unit Lwt.t ->
   ?on_exn:(exn -> unit) ->
-  ?ctx:Net.ctx ->
-  ?mode:Conduit_lwt_unix.server -> t -> unit Lwt.t
+  'cfg ->
+  (_, 'flow) Conduit_lwt.protocol ->
+  ('cfg, 't, 'flow) Conduit_lwt.Service.service -> t -> unit Lwt.t
```

Comment:
Is it possible with this signature to block on the server being ready to receive messages? (I was expecting to see a `unit Lwt.t Lwt.t` or similar somewhere.)

Reply:
I can provide something such as `unit -> unit Lwt_condition.t * unit Lwt.t` if you want to be able to delayed the initialization of the server but I would like to know the opinion of others peoples. Note that such request will fallback to an update of `ocaml-conduit`.

Reply:
I think it's a good idea to do it ; it's a bit painful right-now to write reliable tests which are using cohttp because of this issue (e.g. we need to add an extra retry-logic to make sure the server is ready before starting the tests, see for instance https://github.com/mirage/irmin/blob/master/test/irmin-http/test_http.ml#L61-L82)

Reply:
I agree, this sounds like a useful thing to have